Rocket.Chat/Managers/Model/AuthManager.swift

```diff
@@ -326,7 +326,7 @@ extension AuthManager {
     static func auth(credentials: OAuthCredentials, completion: @escaping MessageCompletion) {
         let params = [
             "oauth": [
-                "credentialToken": credentials.token ?? "",
+                "credentialToken": credentials.token,
```

Comment:
Why this change?

Reply:
https://github.com/RocketChat/Rocket.Chat.iOS/blob/bcf7855ace5a20249dfa51f4431f51b5dd680dc4/Rocket.Chat/Managers/OAuthManager.swift#L15
@rafaelks token is non optional, so this just gives a warning in Xcode, that `?? ""` is not needed.

Reply:
Sounds good! :-)